src/http-api.ts

```diff
@@ -128,6 +128,29 @@ export class HttpApi {
       }
     });
 
+    this.#api.get('/:did/query', async (req, res) => {
+      const options = {} as any;
+      for (const param in req.query as object) {
```

Comment:
Sanity: seem like there is no need to type cast
```suggestion
      for (const param in req.query) {
```